readdirp.js

```diff
@@ -4,8 +4,10 @@ var fs        =  require('graceful-fs')
   , path      =  require('path')
   , minimatch =  require('minimatch')
   , toString  =  Object.prototype.toString
+  , si = typeof setImmediate !== 'undefined' ? setImmediate : function (fn) { setTimeout(fn, 0) }
```

Comment:
Is this here for compatibility with old versions of node? You didn't end up actually using it below on 210